### ISSUE TITLE
Storm does not set constant in property if you set value in the jani file.

### ISSUE THREAD

**Issue by AhmadZafarDGB** [moves-rwth/storm#95]
Getting different results when set constants value using the command line vs setting constants value directly in JANI file.

Set constants values in the command line:
--jani bitcoin-attack.v1.jani --janiproperty --constants 'MALICIOUS=20,CD=6'
Result:
Model checking property "T_MWinMin": T[exp]min=? **[F ((m_len >= 6)** & (m_diff > 0))] ...
Result (for initial states): 3737.134867

Set constants values in the file:
"constants": [
		{
			"name": "MALICIOUS",
			"type": "int",
			"value": 20
		},
		{
			"name": "CD",
			"type": "int",
			"value": 6
		}
	]

 --jani bitcoin-attack.v1.jani --janiproperty
Result:
Model checking property "T_MWinMin": T[exp]min=? **[F ((m_len >= CD)** & (m_diff > 0))] ...
Result (for initial states): 60

Basically, storm uses the default value of the integer (0) in the property when we set the value of constants in JANI file.

**Comment by tquatmann** [comment]
Nice find! Thank you for pointing this out.
